packages/composable-controller/src/ComposableController.test.ts

```diff
@@ -165,19 +168,52 @@ class BazController extends BaseControllerV1<never, BazControllerState> {
   }
 }
 
+type QuxControllerState = {
```

Comment:
Applied: https://github.com/MetaMask/core/pull/4968/commits/74e45f49eedcc37f62f6eb368501a6d5e6f6d2dd

I think this might be enough to avoid confusion, since the V1 mock controllers bar and baz will be removed soon, leaving `FooController`, `QuzController`, and `ControllerWithoutStateChangeEvent`.